Tools/autotest/vehicle_test_suite.py

```diff
@@ -3318,7 +3318,7 @@ def try_symlink_tlog(self):
             os.link(self.logfile, self.buildlog)
         except OSError as error:
             self.progress("OSError [%d]: %s" % (error.errno, error.strerror))
-            self.progress("WARN: Failed to create link: %s => %s, "
+            self.progress("ExcludedError WARN: Failed to create link: %s => %s, "
```

Comment:
We could just say `Problem:` rather than `WARN`?

Reply:
yes, no problem for this.